### PR TITLE
Add a Stimulus controller to configure TinyMCE instances

### PR DESCRIPTION
**Pull request by m-vo**
Fixes one part of #7391

This PR introduces a Stimulus controller for the tinyMCE editor. This way we can gracefully remove and (re-)init editors independent of the document structure.

I tried to keep the changes to the `be_tinyMCE` template to a minimum. If you're only overwriting blocks, nothing needs to be adjusted. The only difference is that we're building a config that's passed on to the controller instead of directly instantiating the editor.

An alternative to passing a JSON string would be to store the config in a global variable as there can only be one at a time anyways (currently).

### CODE REVIEW

File: core-bundle/assets/controllers/tinymce-controller.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+    static values = {
+        config: String
```
Comment (fritzmg) [inline core-bundle/assets/controllers/tinymce-controller.js]: You can use the [`Object`](https://stimulus.hotwired.dev/reference/values#types) type here, then you don't have to execute `JSON.parse()` yourself :)

**Comment by zoglo** [comment]
Tried that one out but it doesn't reinit.
I stumbled upon the same issue of reinitializing the editor on another dom (due to the prefetch rendering)

What I've come up with was this within the backend.js but I'd still need to somehow add the options:

```diff
// contao/core-bundle/assets/backend.js
-// Make MooTools scripts reinitialize themselves
+// Make scripts reinitialize themselves
document.documentElement.addEventListener('turbo:render', () => {
    if (!document.body.mooDomreadyFired) {
        document.body.mooDomreadyFired = true;
        window.fireEvent('domready');
    }
+
+   let rteEditors = [];
+
+   tinymce?.get()?.forEach(editor => {
+       rteEditors.push(editor.id)
+   })
+
+   // Remove all old instances of tinymce
+   tinymce?.remove()
+
+   // Reinitialize
+   rteEditors.forEach(id => {
+       tinymce.init({selector: `#${id}`}) // @ToDo: parse configuration
+   })
});

File: core-bundle/assets/controllers/tinymce-controller.js (hunk 1 above)
Comment (m-vo) [reply core-bundle/assets/controllers/tinymce-controller.js]: Done in 99b99b1

**Comment by m-vo** [comment]
> Tried that one out but it doesn't reinit.

Try again, I had the wrong disconnect line still in there.

**Comment by zoglo** [comment]
That one works :)

**Review by fritzmg — DISMISSED** [review]
Marvelous

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @m-vo.

**Pull request merged**